components/config/featureFlagContext.tsx

```diff
@@ -0,0 +1,31 @@
+"use client"
+
+import { createContext, FC, PropsWithChildren, useContext, useEffect, useState } from "react";
+
+export type FeatureFlags = {
+    staticStageCount: boolean
+}
+
+const featureFlagsContext = createContext<FeatureFlags>({ staticStageCount: true });
+export const useFeatureFlags = () => useContext(featureFlagsContext);
+
+export const FeatureFlagProvider: FC<PropsWithChildren> = ({ children }) => {
+
+    const [featureFlags, setFeatureFlags] = useState<FeatureFlags>({
+        staticStageCount: true
+    });
```

Comment:
make the default fridge a single variable

Reply:
Added